app/src/pages/005_Create_Member.py

```diff
@@ -0,0 +1,52 @@
+import streamlit as st
+from datetime import datetime
+import logging
+
```

Comment:
See below suggestion
```suggestion
import requests
```